wiki/Shortcut_Key_Reference/en.md

```diff
@@ -1,179 +1,233 @@
+- `F1`: [Mod](/wiki/Game_Modifiers) selection menu
+- `F2`: Randomly select a map
+- `Shift` + `F2`: Undo random map selection
+- `F3`: Beatmap options
+- `F5`: Reprocess beatmaps (may display a dialog)
+- `Shift` + `Delete`: Delete selected map
+- `Ctrl` + `1`: Switch to [osu!](/wiki/osu!)
+- `Ctrl` + `2`: Switch to [osu!taiko](/wiki/osu!taiko)
+- `Ctrl` + `3`: Switch to [osu!catch](/wiki/osu!catch)
+- `Ctrl` + `4`: Switch to [osu!mania](/wiki/osu!mania)
+- `Ctrl` + `Enter`: Play current map with [Auto](/wiki/Auto) mod
+- `Ctrl` + `Shift` + `Enter`: Play current map with [Cinema](/wiki/Cinema) mod
+- `Ctrl` + `A`: Toggle Auto mod
+- `Ctrl` + `F3`: Increase osu!mania speed (only in osu!mania mode)
+  - alias: `Ctrl` + `+`
+- `Ctrl` + `F4`: Decrease osu!mania speed (only in osu!mania mode)
+  - alias: `Ctrl` + `-`
+- `Ctrl` + `Backspace`: Delete last word (when using search)
+- `Up`: Move selector one beatmap up
+- `Down`: Move selector one beatmap down
+- `Left`: Change selected beatmapset one up
+- `Right`: Change selected beatmapset one down
+- `Page Up`: Scroll up
+- `Page Down`: Scroll down
+- `Enter`: Select selection or play selection
+- `Shift` + `Left Arrow`: Change selected group one up
+- `Shift` + `Right Arrow`: Change selected group one down
+- `Shift` + `Enter`: Toggle selected group
+
+### Game Modifiers
+
+_See also: [Game Modifiers](/wiki/Game_Modifiers)._
+
+**Notes:**
+
+- The key references below assumes that none of the mods are selected.
+- Some mods may require the player to press the key twice.
+
+- `Q`: [Easy](/wiki/Easy_mod)
+- `W`: [No Fail](/wiki/No_Fail)
+- `E`: [Half Time](/wiki/Half_Time)
+
+---
+
+- `A`: [Hard Rock](/wiki/Hard_Rock)
+- `S`: [Sudden Death](/wiki/Sudden_Death)
+- `S`, then `S`: [Perfect](/wiki/Perfect)
+- `D`: [Double Time](/wiki/Double_Time)
+- `D`, then `D`: [Nightcore](/wiki/Nightcore)
+- `F`: [Hidden](/wiki/Hidden) (osu!standard, osu!taiko, and osu!catch)
+  - this will toggle [Fade In](/wiki/Fade_In) for osu!mania
+- `F`, then `F`: [Hidden](/wiki/Hidden) (osu!mania only)
+- `G`: [Flashlight](/wiki/Flashlight)
+
+---
+
+- `X`: [Relax](/wiki/Relax)
+- `C`: [Auto Pilot](/wiki/Auto_Pilot)
+- `V`: [Spun Out](/wiki/Spun_Out)
+- `B`: Auto
+- `B`, then `B`: Cinema
+
+---
+
+- `3`: [3K](/wiki/3K) (osu!mania only)
+- `4`: [4K](/wiki/4K) (osu!mania only)
+- `5`: [5K](/wiki/5K) (osu!mania only)
+- `6`: [6K](/wiki/6K) (osu!mania only)
+- `7`: [7K](/wiki/7K) (osu!mania only)
+- `8`: [8K](/wiki/8K) (osu!mania only)
+- `9`: [9K](/wiki/9K) (osu!mania only)
+
+## Play Mode
+
+- `+` or `-`: Adjust local offset (hold `Alt` to change the precision)
+- `Esc`: Pause song (or quit if Auto or Cinema mod is used)
+- `Space`: Skip opening cut-scene
+- `Tab`: Toggle scoreboard
+- `Shift` + `Tab`: Toggle in-game interface
+- `F3` or `F4`: Adjust speed for osu!mania (only works near the beginning of playing)
+- `Ctrl` + `R` or `` ` `` (hold briefly): Quick retry the current map (autoskips the cut-scene upon retry)
```

Comment:
cut-scene -> intro

I think those skip-able sections are called intro and outro for starting and ending. I am guessing it so take my idea with a grain of salt.